src/components/Game.js

```diff
@@ -53,25 +73,45 @@ const Game = () => {
   return (
     <div className="game">
```

Comment:
I think you may need to refactor your BEM names a bit?

`game` would be the block level, making everything below it names like:

- `game__info`
- `game__title`
- `game__player`
- `game__player--o`
- `game__player--x`
- `game__content`

... and so on!